modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/processors/hdbschema/HDBSchemaDropProcessor.java

```diff
@@ -26,13 +26,13 @@
   private static final Logger logger = LoggerFactory.getLogger(HDBSchemaDropProcessor.class);
 
   public void execute(Connection connection, XSKDataStructureHDBSchemaModel hdbSchema) throws SQLException {
-    logger.info("Processing Drop Schema: " + hdbSchema.getName());
+    logger.info("Processing Drop Schema: " + hdbSchema.getSchema());
 
     ISqlDialect dialect = SqlFactory.deriveDialect(connection);
     if (!(dialect.getClass().equals(HanaSqlDialect.class))) {
       throw new IllegalStateException(String.format("%s does not support Schema", dialect.getDatabaseName(connection)));
     } else {
-      String sql = "DROP SCHEMA " + hdbSchema.getName();
+      String sql = "DROP SCHEMA " + hdbSchema.getSchema();
```

Comment:
Use escapeArtifactName method in order to take into consideration if we have caseSensitive flag set as true.
`String schemaName = XSKHDBUtils.escapeArtifactName(connection, hdbSchema.getSchema());`